source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -771,7 +771,7 @@ def merge_responses(responses, debug):
             fp
             for fp in r.feed_publishers
             if fp.id not in initial_feed_publishers
-            and (debug or all('to_delete' not in j.tags for j in r.journeys))
+            and (debug or len(responses) == 1 or all('to_delete' not in j.tags for j in r.journeys))
```

Comment:
this won't work when every proto_reponse return kraken contain a journey tagged as 'to_delete'... 

I'm stuck...

Reply:
there is no `any` in python as `any_of` in C++?